sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/persons/PersonDirectorySteps.java

```diff
@@ -123,6 +123,12 @@ public PersonDirectorySteps(
           webDriverHelpers.clickOnWebElementBySelector(getByPersonUuid(personUuid));
         });
 
+    When(
+        "I click on first person in person directory",
```

Comment:
we might need to get rid of this and create a new person for it because if the first person is dead/buried we won't be able to perform this action.
Maybe we can keep if, but we need first to apply a valid filter

Reply:
OK, I added steps that filter for persons who are alive beforehand.